pages/signup/index.tsx

```diff
@@ -100,17 +79,32 @@ const SignUpPage = ({ t }: PageProps): React.ReactElement => {
         .signup(formValues.email, formValues.password)
         .then((response: FirebaseUserCredential) => {
           if (response.user) {
-            return createUser({
-              variables: {
-                input: {
-                  id: response.user.uid,
-                  ...addUserInputTransformer(formValues),
+            const input = {
+              ...transformSignupFormValuesToTalentInput(formValues),
```

Comment:
why don't we put this in the addTalent helper?
To me, it makes more sense if the signup component delivers the formValues that are actually provided by the user. Then, the resolver would receive the UserInput or PartialTalent or how we  want to call it. And then the helper would do this transformation (`transformSignupFormValuesToTalentInput`).

Reply:
addTalent expects a TalentInput. This way, we don't need to multiply graphql types. So, I'd want to keep i this way.

Reply:
Okay, fine for me. I don't like the SignUp Page having to do this transformation. I don't think it belongs here. But it is kind of a tradeoff. So fine for me.

Reply:
But if addTalent expects a complete TalentInput where else would it belong? I guess one could move queries and mutations out of the components into their own files and then do the transformations there. However, graphql documentation is pretty explicit that they consider it best practice to define the queries and mutations directly where they are used.